src/app.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi, { SwaggerUiOptions } from 'swagger-ui-express';
+
+import { healthCheckRoute } from './routes/api/health-check.js';
```

Comment:
I need to use .js extension in my import because I use "module":"nodenext" in my tsconfig.json and "type":"module" in package.json. Should I swithch back to commonJS?

Reply:
In your homework there is no information about which module in tsconfig to use, so it is ok